CHANGELOG.markdown

```diff
@@ -21,6 +21,17 @@
     swapped = iso Swap.swap Swap.swap
   #endif
   ```
+* `FunctorWithIndex`, `FoldableWithIndex` and `TraversableWithIndex` type classes
+  are extracted into new package `traversable-with-index` (TODO: to be confirmed, add hackage link).
+
+  The `imapped`, `ifolded` and `itraversed` methods are now normal functions.
+  If you are not defining these methods in your instances,
+  you don't need to change your definitions.
+
+  Beware: the `optics-core` package defines similar classes, and will also
```

Comment:
It might be helpful to mention which versions of `optics-core` define duplicates of these classes.

Reply:
Good point. We don't know yet.

Reply:
I think that <0.4 is a good guess though.